articles/sendgrid-dotnet-how-to-send-email.md

```diff
@@ -149,7 +149,83 @@ The following examples show how to send a message using the Web API.
             }
         }
     }
+    
+## How to: Send email from ASP .NET Core API using MailHelper class
 
+The below example can be used to send a single email to multiple persons from ASP .NET Core api using the `MailHelper` class of `SendGrid.Helpers.Mail` namespace. For this example we are using ASP .NET Core 1.0. 
+
+In this example the Api key has been stored in the `appsettings.json` file  which can be overridden from Azure portal as shown in above examples.
+
+The contents of `appsettings.json` file should look like below  -
+
+    {
+       "Logging": {
+       "IncludeScopes": false,
+       "LogLevel": {
+       "Default": "Debug",
+       "System": "Information",
+       "Microsoft": "Information"
+         }
+       },
+     "SENDGRID_API_KEY": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+
+As first step we need to add the below code in Startup.cs file of the .NET Core api project. This is required so that we can access the `SENDGRID_API_KEY` from the `appsettings.json` file by using dependency injection in the api controller. The `IConfiguration` interface can be injected at the constructor of controller after adding it in the `ConfigureServices` method below. The content of `Startup.cs` file looks like below after adding the required code -
```

Comment:
As first step we > First, we

in Startup.cs file > in the `Startup.cs` file

.NET Core api > .NET Core API

api controller > API controller

constructor of controller > constructor of the controller

looks like below after adding the required code - > looks like the following after adding the required code:

Reply:
ok i will modify it . Thanks for the input :)